docs/plugins/mongodb.md

```diff
@@ -0,0 +1,40 @@
+# `mongodb` plugin
+
+The `mongodb` plugin consumes [MongoDB](https://www.mongodb.com/) log entries from the local filesystem and outputs parsed entries.
+
+## Configuration Fields
+
+| Field | Default | Description |
+| --- | --- | --- |
+| `source` | `file` | Use this field to specify where your logs are coming from. When choosing the 'file' option, the agent reads in logs from the log paths specified below.  When choosing the 'Kubernetes' options, the agent reads logs from /var/log/containers based on the Pod and Container specified below. |
+| `cluster_name` | `""`  | Cluster Name to be added to a resource label |
```

Comment:
This is a "friendly" name, useful for filtering between clusters when in a multi cluster environment.

Reply:
changed description to better reflect that